integration_test/docker_test/docker_test.go

```diff
@@ -175,12 +175,12 @@ func testCases(t *testing.T) {
 			eventSql := "select anonymous_id, user_id from dev_integration_test_1.identifies limit 1;"
 			_ = db.QueryRow(eventSql).Scan(&myEvent.anonymousID, &myEvent.userID)
 			return myEvent.anonymousID == "anonymousId_1"
-		}, time.Minute, 10*time.Millisecond)
+		}, 2*time.Minute, 10*time.Millisecond)
```

Comment:
why do we need to increase this?

Reply:
It was taking a bit more time to upload since we are doing things async rather than synchronously

Reply:
Can we try to revert with the updated configs? is it still failing? If so, we may modify configuration options in the test itself